opennms-base-assembly/src/main/filtered/etc/poller-configuration.xml

```diff
@@ -213,6 +213,12 @@
          <parameter key="rrd-base-name" value="pop3"/>
          <parameter key="ds-name" value="pop3"/>
       </service>
+      <service name="PTP" interval="300000" user-defined="false" status="on">
+         <pattern><![CDATA[^PTP-(?<clockPort>.*)-(?<portState>.*)$]]></pattern>
```

Comment:
```suggestion
         <pattern><![CDATA[^PTP-.*$]]></pattern>
```

Reply:
Allow a user to create any service name with PTP-* and apply the service config section to it. Remove the regexp group extraction for configs.